estate/models/estate_property_offer.py

```diff
@@ -0,0 +1,70 @@
+from odoo import fields, models ,api
+from datetime import datetime, timedelta 
+from odoo.exceptions import UserError,ValidationError
+
+class EstatePropertyOffer(models.Model):
+    _name = "estate.property.offer"
+    _description = "Estate Property Offers"
+    
+    
+    price = fields.Float()
+    status = fields.Selection(
+        selection=[
+            ('Accepted', 'Accepted'),
+            ('Refused', 'Refused'),
+        ],
+        string="Status"
+    )
+    partner_id = fields.Many2one('res.partner', required=True)
+    property_id = fields.Many2one('estate.property', required=True)
+    validity = fields.Integer(default=7)
+    deadline_date = fields.Date(compute="_compute_validity", inverse="_inverse_validity",store=True)
+    created_date = fields.Date(default=lambda self: datetime.today())
+    accepted = fields.Boolean(default=False)
+
+    @api.depends("validity", "created_date")
+    def _compute_validity(self):
+        for record in self:
+            if record.created_date:
+                record.deadline_date = record.created_date + timedelta(days=record.validity)
+            else:
+                record.deadline_date = False
+                
+    @api.depends("created_date", "deadline_date")
```

Comment:
No need for this i think we can keep the generic behavior of inverse method that reflects the changes after saving the records in our usecase.